src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx

```diff
@@ -0,0 +1,92 @@
+* **Account**: When you sign up for New Relic your organization is automatically assigned one account. An account can have many sources of data reporting to it. For example, a single account could have data reporting from our infrastructure agent, an APM agent, and other integrations. All data reported to New Relic requires an account ID, which tells New Relic which account that data belongs in. That ID is also used for some account-specific tasks, like making API calls.
+* **User**: In your organization, your New Relic users are granted access to specific accounts that are relevant to their duties and responsibilities. To manage users' access to accounts, you grant groups access to specific roles on specific accounts. Our user management system allows you to create the user access you need, whether that's a relatively simple setup with just a few roles across a few accounts, or a complex one with many roles across many accounts. Learn more about user management.
+
+## What account structure is right for me? [#account-structure]
+
+Signing up for New Relic automatically generates an organization. An organization contains one account unless you manually create more. One account can be perfect for many companies and it might not be necessary for you to add more. 
+
+Some indications that one account is right for you:
+
+* You have 200 employees or less and you don't anticipate growing too much
+* You aren't ingesting vast amounts of data. If you're not sure, use [our spreadsheet](https://newrelic.com/blog/nerdlog/estimate-data-cost) to estimate the amount of data you might send to New Relic
+* You expect to send a moderate amount of alerts. If you're not sure, review our [alerting rules and limits](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts/)
+
+If these don't apply to you, it might be good to consider a multiple account structure. 
+
+### Use multiple accounts to separate testing and production environments [#environments]
+
+If you create multiple accounts you can observe data across multiple environments. With multiple accounts, you can monitor changes before they go to production. If you’re updating your application weekly, or even daily, it is important that you have a strong testing environment in place so you can make sure each deployment is smooth and error free.
+
+
+<img
+    title="multiple environments"
+    alt="a diagram demonstrating how to use New Relic with multiple environments"
+    src={accountsMultipleAccountEnvironments}
+/>
+
+### Use multiple accounts to separate projects with unique data sets [#data-sets]
+
+If you manage a payment provider that works in multiple countries, you might need to manage different currencies and different tax regulations. Or, you could manage a social media site and you might need to comply with different data collection policies in different regions. In both those situations, it is a good idea to create multiple accounts. 
```

Comment:
Can you unspool more literally what these examples have in common so people can extract how it applies to them if they don't fit these two examples. For example, maybe clarifying that multiple accounts is good if you have competing laws that need to be separated (idk how to word it but thats just me riffing)